draft-ietf-tls-tls13.md

```diff
@@ -2976,7 +3002,7 @@ MUST be the first PSK listed in the client's "pre_shared_key" extension.
 For PSKs provisioned via NewSessionTicket, a server MUST validate that
 the ticket age for the selected PSK identity (computed by subtracting
 ticket_age_add from PskIdentity.obfuscated_ticket_age modulo 2^32)
-is within a small tolerance of the
+is within a small tolerance (a small number of seconds) of the
```

Comment:
Is it better for us to make up numbers or implementers to make up numbers?

Reply:
Implementers will manage.  FWIW, NSS kicks the can even further down the road.

Reply:
Okay.